kotlin/android/app/src/main/java/dev/firezone/android/features/applink/ui/AppLinkHandlerActivity.kt

```diff
@@ -27,11 +27,13 @@ class AppLinkHandlerActivity : AppCompatActivity(R.layout.activity_app_link_hand
         viewModel.actionLiveData.observe(this) { action ->
             when (action) {
                 is AppLinkViewModel.ViewAction.AuthFlowComplete -> {
-                    // Continue with onboarding
+                    // TODO: Continue starting the session showing sessionFragment
```

Comment:
@pratikvelani This is where I left off. We need to show the session fragment instead of the `AUTH Complete` TextView here.